components/ui/Footer.js

```diff
@@ -26,21 +35,16 @@ export const Footer = () => {
 
       {/* TASK - React 1 week 1 */}
       {/* Add a new list item for LINKEDIN */}
+
       <div className={styles.footerLinks}>
         <h3>Follow us</h3>
+        {/* added linkedn and used map method */}
         <ul className={styles.footerList}>
-          <li>
-            <a href="https://facebook.com">Facebook</a>
-          </li>
-          <li>
-            <a href="https://instagram.com">Instagram</a>
-          </li>
-          <li>
-            <a href="https://tiktok.com">Tiktok</a>
-          </li>
-          <li>
-            <a href="https://google.com">On the streets at night</a>
-          </li>
+          {socialMedia.map((item) => (
+            <li key={item.id}>
```

Comment:
If you follow the above advice.
```suggestion
            <li key={item.name}>
```